content/posts/2025-01-23-cli_whats_new_1.8.md

```diff
@@ -0,0 +1,229 @@
+---
+layout: :theme/post
+title: "Network Observability On Demand 1.8 Update"
+description: Command line interface improvements in version 1.8
+tags: CLI,Monitoring,Troubleshooting
+authors: [jpinsonneau]
+---
+
+# Network Observability On Demand 1.8 Update
+
+```
+------------------------------------------------------------------------
+         _  _     _       _                       ___ _    ___
+        | \| |___| |_ ___| |__ ___ ___ _ ___ __  / __| |  |_ _|
+        | .' / -_)  _/ _ \ '_ (_-</ -_) '_\ V / | (__| |__ | | 
+        |_|\_\___|\__\___/_.__/__/\___|_|  \_/   \___|____|___|
+
+------------------------------------------------------------------------
+```
+
+Since we [introduced the Network Observability CLI](./2024-07-25-cli.md), numerous features have been added. This article will cover the improvements in version 1.8 and provide some concrete examples.
+
+## New Options Available
+This update adds several options to the CLI, covering more scenarios and enabling scripting on top of the tool.
+
+### Operate in the Background
+The `--background` option allows you to start a flow or packet capture without connecting your terminal to the collector pod. This enables you to let the capture run in the background while you work on something else. You can check the capture's progress using the `follow` command and copy the collector output locally using the `copy` command. Once the job is complete, you can `stop` or `cleanup` everything.
+
+```sh
+oc netobserv flows --background # Run a flow capture in the background
+
+oc netobserv follow             # Show current capture progression
+
+oc netobserv stop               # Stop the capture keeping the collector running
+
+oc netobserv copy               # Copy the output
+
+oc netobserv cleanup            # Remove all
+```
+
+### Customizable Namespace
+You can now customize the capture namespace using the `NETOBSERV_NAMESPACE` environment variable. When the CLI starts, it automatically checks if this namespace exists and will stop if it finds any conflict with a pending capture. This is particularly useful if you want to run captures in parallel.
+
+```sh
+NETOBSERV_NAMESPACE=my_ns oc netobserv [flows|packets|metrics|follow|stop|copy|cleanup]
+```
+
+### Subnets Labelling
+The tool can now read configurations from `cluster-config-v1` and `network` to identify **Machine**, **Pods**, and **Services** subnets using the `--get-subnets` option. This will automatically add `SrcSubnetLabel` and `DstSubnetLabel` to your flows.
```

Comment:
Maybe mention `in OpenShift` ? the `network` CR only exists in openshift

Reply:
Sure, I can put that in the title and the text:

```suggestion
The tool can now read Openshift configurations from `cluster-config-v1` and `network` to identify **Machine**, **Pods**, and **Services** subnets using the `--get-subnets` option. This will automatically add `SrcSubnetLabel` and `DstSubnetLabel` to your flows.
```